packaging/python_wheel/setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from setuptools import setup, Distribution
+
+VERSION = '0.0.0'
```

Comment:
Need a good way to get version information, either:
 - through python
 - have cmake put it in this file